helix-front/server/config.example.ts

```diff
@@ -56,3 +56,9 @@ export const TOKEN_EXPIRATION_KEY: string = 'expires';
  * This key can be null or a string.
  */
 export const APP_INSIGHTS_CONNECTION_STRING: string | null = null;
+
+/**
+ * An optional proxy url
+ *
+ */
+export const PROXY_URL: string = 'www.example.com';
```

Comment:
I think these values from [PR 9 of helix-ui](https://linkedin.ghe.com/multiproduct/helix-ui/pull/9/files) need to go in this file?
![CleanShot 2023-03-22 at 18 10 38@2x](https://user-images.githubusercontent.com/6743796/227073212-a8a4351e-6368-4de3-b414-84bc68ad076c.png)